android/src/main/java/io/customer/reactnative/sdk/CustomerioReactnativePackage.kt

```diff
@@ -1,14 +1,15 @@
-package com.customerioreactnative
+package io.customer.reactnative.sdk
```

Comment:
Will this change affect the react bridge file ?

Reply:
No as, we didn't change the bridge name. Thanks for checking.